packages/react-scripts/template/README.md

```diff
@@ -1407,6 +1411,40 @@ Learn more about React Storybook:
 * [Documentation](https://storybook.js.org/basics/introduction/)
 * [Snapshot Testing UI](https://github.com/storybooks/storybook/tree/master/addons/storyshots) with Storybook + addon/storyshot
 
+### Getting Started with Styleguidist
+
+Styleguidist combines of a style guide, where all your components are presented on a single page with their props documentation and usage examples, with an environment for developing components in isolation, similar to Storybook. In Styleguidist you write examples in Markdown, where each code snippet is rendered as a live editable playground.
+
+First install Styleguidist and peer dependencies from npm:
+
+```sh
+npm install --save-dev react-styleguidist webpack
+```
+
+Then, add these scripts to your `package.json`:
+
+```sh
+{
+  "scripts": {
```

Comment:
User naïvely following this instruction will replace the whole `scripts` section.